src/components/Form/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+
+import "./style.css";
+
+export default ({ getUserInfo }) => {
+  const [username, setUsername] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await getUserInfo(username);
+      // Clear input box on submit
+      e.target.childNodes[1].value = "";
+    } catch (err) {
+      console.warn(err);
+    }
```

Comment:
would be nice to search for another person after first search🤓

Reply:
Yeah lmao.
We added that to the future feautures... would've perhaps been more practical than refreshing the screen 🤣